openpype/lib/transcoding.py

```diff
@@ -120,6 +120,10 @@ def get_oiio_info_for_input(filepath, logger=None, subimages=False):
 
     output = []
     for subimage_lines in subimages_lines:
+        # First line of oiiotool for xml format is "Reading path/to/file.ext".
```

Comment:
That's why this condition is there https://github.com/ynput/OpenPype/pull/6129/files#diff-69bbdd557800051372da1b473a3b228d28c24f08c74c677c0b4dd81002056f50R104

Reply:
Here is a full job report from Deadline:
```
=======================================================
Error
=======================================================
Error: Renderer returned non-zero error code, 1. Check the log for more information.
   at Deadline.Plugins.PluginWrapper.RenderTasks(Task task, String& outMessage, AbortLevel& abortLevel)

=======================================================
Type
=======================================================
RenderPluginException

=======================================================
Stack Trace
=======================================================
   at Deadline.Plugins.SandboxedPlugin.d(DeadlineMessage bgq, CancellationToken bgr)
   at Deadline.Plugins.SandboxedPlugin.RenderTask(Task task, CancellationToken cancellationToken)
   at Deadline.Slaves.SlaveRenderThread.c(TaskLogWriter ajv, CancellationToken ajw)

=======================================================
Log
=======================================================
2024-01-15 14:36:05:  0: Loading Job's Plugin timeout is Disabled
2024-01-15 14:36:05:  0: SandboxedPlugin: Render Job As User disabled, running as current user 'tokejepsen'
2024-01-15 14:36:07:  0: Executing plugin command of type 'Initialize Plugin'
2024-01-15 14:36:07:  0: INFO: Executing plugin script 'C:\ProgramData\Thinkbox\Deadline10\workers\DESKTOP-969549J\plugins\65a5427f2e5e77f3b44c5516\Ayon.py'
2024-01-15 14:36:07:  0: INFO: Plugin execution sandbox using Python version 3
2024-01-15 14:36:07:  0: INFO: Single Frames Only: False
2024-01-15 14:36:07:  0: INFO: About: Ayon Plugin for Deadline
2024-01-15 14:36:07:  0: INFO: The job's environment will be merged with the current environment before rendering
2024-01-15 14:36:07:  0: Done executing plugin command of type 'Initialize Plugin'
2024-01-15 14:36:07:  0: Start Job timeout is disabled.
2024-01-15 14:36:07:  0: Task timeout is disabled.
2024-01-15 14:36:07:  0: Loaded job: Publish - renderAnimationMain (65a5427f2e5e77f3b44c5516)
2024-01-15 14:36:07:  0: Executing plugin command of type 'Start Job'
2024-01-15 14:36:07:  0: DEBUG: S3BackedCache Client is not installed.
2024-01-15 14:36:07:  0: INFO: Executing global asset transfer preload script 'C:\ProgramData\Thinkbox\Deadline10\workers\DESKTOP-969549J\plugins\65a5427f2e5e77f3b44c5516\GlobalAssetTransferPreLoad.py'
2024-01-15 14:36:07:  0: INFO: Looking for legacy (pre-10.0.26) AWS Portal File Transfer...
2024-01-15 14:36:07:  0: INFO: Looking for legacy (pre-10.0.26) File Transfer controller in C:/Program Files/Thinkbox/S3BackedCache/bin/task.py...
2024-01-15 14:36:07:  0: INFO: Could not find legacy (pre-10.0.26) AWS Portal File Transfer.
2024-01-15 14:36:07:  0: INFO: Legacy (pre-10.0.26) AWS Portal File Transfer is not installed on the system.
2024-01-15 14:36:07:  0: INFO: Executing global job preload script 'C:\ProgramData\Thinkbox\Deadline10\workers\DESKTOP-969549J\plugins\65a5427f2e5e77f3b44c5516\GlobalJobPreLoad.py'
2024-01-15 14:36:07:  0: PYTHON: *** GlobalJobPreload start ...
2024-01-15 14:36:07:  0: PYTHON: >>> Getting job ...
2024-01-15 14:36:07:  0: PYTHON: >>> Injecting render job id ...
2024-01-15 14:36:07:  0: PYTHON: >>> Dependency IDs: System.String[]
2024-01-15 14:36:07:  0: INFO: Setting Process Environment Variable RENDER_JOB_IDS to 65a5427f2e5e77f3b44c5515
2024-01-15 14:36:07:  0: PYTHON: >>> Injection end.
2024-01-15 14:36:07:  0: Done executing plugin command of type 'Start Job'
2024-01-15 14:36:07:  0: Plugin rendering frame(s): 0
2024-01-15 14:36:08:  0: Executing plugin command of type 'Render Task'
2024-01-15 14:36:08:  0: INFO: Stdout Redirection Enabled: True
2024-01-15 14:36:08:  0: INFO: Stdout Handling Enabled: True
2024-01-15 14:36:08:  0: INFO: Popup Handling Enabled: False
2024-01-15 14:36:08:  0: INFO: Using Process Tree: True
2024-01-15 14:36:08:  0: INFO: Hiding DOS Window: True
2024-01-15 14:36:08:  0: INFO: Creating New Console: False
2024-01-15 14:36:08:  0: INFO: Running as user: tokejepsen
2024-01-15 14:36:08:  0: INFO: Executable: "C:\Users\tokejepsen\AppData\Local\ynput\ayon\app\AYON 1.0.0-rc.2\ayon.exe"
2024-01-15 14:36:08:  0: INFO: Argument: --headless publish "{root[work]}/demo_Commercial/shots/sh010/work/animation/renders/maya/com_sh010_workfileAnimation_v010/Main/renderAnimationMain_metadata.json" --targets deadline --targets farm
2024-01-15 14:36:08:  0: INFO: Full Command: "C:\Users\tokejepsen\AppData\Local\ynput\ayon\app\AYON 1.0.0-rc.2\ayon.exe" --headless publish "{root[work]}/demo_Commercial/shots/sh010/work/animation/renders/maya/com_sh010_workfileAnimation_v010/Main/renderAnimationMain_metadata.json" --targets deadline --targets farm
2024-01-15 14:36:08:  0: INFO: Startup Directory: "C:\Users\tokejepsen\AppData\Local\ynput\ayon\app\AYON 1.0.0-rc.2"
2024-01-15 14:36:08:  0: INFO: Process Priority: BelowNormal
2024-01-15 14:36:08:  0: INFO: Process Affinity: default
2024-01-15 14:36:08:  0: INFO: Process is now running
2024-01-15 14:36:10:  0: STDOUT: WARNING:pyblish.plugin:Path already registered: C:\Users\tokejepsen\AppData\Local\Ynput\AYON\addons\slack_1.0.1\ayon_slack\plugins\publish
2024-01-15 14:36:10:  0: STDOUT: WARNING:pyblish.plugin:Path already registered: C:\Users\tokejepsen\OpenPype\openpype\modules\deadline\plugins\publish
2024-01-15 14:36:10:  0: STDOUT: WARNING:pyblish.plugin:Path already registered: C:\Users\tokejepsen\OpenPype\openpype\modules\timers_manager\plugins\publish
2024-01-15 14:36:10:  0: STDOUT: >>> [ Setting OCIO environment to config path: C:\Users\tokejepsen\AppData\Local\Ynput\AYON\addons\ayon_ocio_1.0.0\ayon_ocio\configs\OpenColorIOConfigs\aces_1.2\config.ocio ] 
2024-01-15 14:36:10:  0: STDOUT: >>> [ Running publish ... ] 
2024-01-15 14:36:11:  0: STDOUT: ERROR:pyblish.plugin:Skipped: "submit_maya_deadline" (No module named 'maya')
2024-01-15 14:36:11:  0: STDOUT: *** WRN: >>> { filter_pyblish_plugins }: [ Couldn't find plugin 'IntegrateSlackAPI' settings under settings category 'slack' ] 
2024-01-15 14:36:11:  0: STDOUT: ERROR:pyblish.plugin:Skipped: "submit_maya_deadline" (No module named 'maya')
2024-01-15 14:36:11:  0: STDOUT: *** WRN: >>> { filter_pyblish_plugins }: [ Couldn't find plugin 'IntegrateSlackAPI' settings under settings category 'slack' ] 
2024-01-15 14:36:11:  0: STDOUT: INFO:pyblish.CollectCurrentContext:Collected project context
2024-01-15 14:36:11:  0: STDOUT: Project: demo_Commercial
2024-01-15 14:36:11:  0: STDOUT: Asset: /shots/sh010
2024-01-15 14:36:11:  0: STDOUT: Task: animation
2024-01-15 14:36:11:  0: STDOUT: INFO:pyblish.CollectMachineName:Machine name: DESKTOP-969549J
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:*** AYON addons initialization.
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] OCIODistModule
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] ThirdPartyDistAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] SlackIntegrationAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[ ] AssetReporterAction
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] AvalonModule
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[ ] ClockifyModule
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] DeadlineModule
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] JobQueueModule
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] LauncherAction
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[ ] LogViewModule
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[ ] MusterModule
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[ ] ProjectManagerAction
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] PythonInterpreterAction
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[ ] RoyalRenderModule
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[ ] LocalSettingsAction
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[ ] SettingsAction
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] TimersManager
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] WebServerModule
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] AfterEffectsAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] BlenderAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] CelactionAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] FusionAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] HieroAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] HoudiniAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] MaxAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] MayaAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] NukeAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] PhotoshopAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] ResolveAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[ ] StandAlonePublishAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] SubstanceAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] TrayPublishAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] TVPaintAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] UnrealAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:[X] WebpublisherAddon
2024-01-15 14:36:11:  0: STDOUT: DEBUG:ModulesManager:Has 26 enabled modules.
2024-01-15 14:36:11:  0: STDOUT: DEBUG:pyblish.CollectAnatomyObject:Anatomy object collected for project "demo_Commercial".
2024-01-15 14:36:11:  0: STDOUT: DEBUG:pyblish.CollectRenderedFiles:Getting root setting for project "demo_Commercial"
2024-01-15 14:36:11:  0: STDOUT: DEBUG:pyblish.CollectRenderedFiles:Anatomy roots: {'work': P:/PROJECTS}
2024-01-15 14:36:11:  0: STDOUT: DEBUG:pyblish.CollectRenderedFiles:Setting session using data from file
2024-01-15 14:36:11:  0: STDOUT: DEBUG:pyblish.CollectRenderedFiles:  - processing instance for renderAnimationMain_beauty
2024-01-15 14:36:11:  0: STDOUT: DEBUG:pyblish.CollectRenderedFiles:Filling stagingDir with root to: P:/PROJECTS/demo_Commercial/shots/sh010/work/animation/renders/maya/com_sh010_workfileAnimation_v010/Main
2024-01-15 14:36:11:  0: STDOUT: DEBUG:GlobalServerAPI:Executing [GET] projects/demo_Commercial
2024-01-15 14:36:11:  0: STDOUT: DEBUG:urllib3.connectionpool:http://localhost:5000 "GET /api/projects/demo_Commercial HTTP/1.1" 200 6079
2024-01-15 14:36:11:  0: STDOUT: DEBUG:GlobalServerAPI:Response <RestApiResponse [200]>
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectContextEntities:Collected Project "{'_id': 'demo_Commercial', 'name': 'demo_Commercial', 'schema': 'openpype:project-3.0', 'type': 'project', 'config': {'apps': [{'name': 'maya/2023'}, {'name': 'maya/2022'}, {'name': 'nuke/15-0'}, {'name': 'hiero/15-0'}], 'roots': {'work': {'linux': '/mnt/share/projects', 'darwin': '/Volumes/projects', 'windows': 'P:/PROJECTS'}}, 'templates': {'defaults': {'frame': '{frame:0>{@frame_padding}}', 'version': 'v{version:0>{@version_padding}}', 'frame_padding': 4, 'version_padding': 3}, 'others': {'work_unreal': {'file': '{project[code]}_{folder[name]}.{ext}', 'folder': '{root[work]}/{project[name]}/unreal/{task[name]}', 'path': '{root[work]}/{project[name]}/unreal/{task[name]}/{project[code]}_{folder[name]}.{ext}'}, 'publish_online': {'file': '{originalBasename}<.{@frame}><_{udim}>.{ext}', 'folder': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/{subset}/{@version}', 'path': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/{subset}/{@version}/{originalBasename}<.{@frame}><_{udim}>.{ext}'}, 'publish_render': {'file': '{project[code]}_{folder[name]}_{subset}_{@version}<_{output}><.{@frame}>.{ext}', 'folder': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/{subset}/{@version}', 'path': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/{subset}/{@version}/{project[code]}_{folder[name]}_{subset}_{@version}<_{output}><.{@frame}>.{ext}'}, 'publish_source': {'file': '{originalBasename}.{ext}', 'folder': '{root[work]}/{originalDirname}', 'path': '{root[work]}/{originalDirname}/{originalBasename}.{ext}'}, 'publish_maya2unreal': {'file': '{subset}_{@version}<_{output}><.{@frame}>.{ext}', 'folder': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}', 'path': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/{subset}_{@version}<_{output}><.{@frame}>.{ext}'}, 'publish_simpleUnrealTexture': {'file': '{originalBasename}_{@version}.{ext}', 'folder': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/{@version}', 'path': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/{@version}/{originalBasename}_{@version}.{ext}'}, 'publish_simpleUnrealTextureHero': {'file': '{originalBasename}.{ext}', 'folder': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/hero', 'path': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/hero/{originalBasename}.{ext}'}}, 'work': {'file': '{project[code]}_{folder[name]}_{task[name]}_{@version}<_{comment}>.{ext}', 'folder': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/work/{task[name]}', 'path': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/work/{task[name]}/{project[code]}_{folder[name]}_{task[name]}_{@version}<_{comment}>.{ext}'}, 'publish': {'file': '{project[code]}_{folder[name]}_{subset}_{@version}<_{output}><.{@frame}><_{udim}>.{ext}', 'folder': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/{subset}/{@version}', 'path': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/{subset}/{@version}/{project[code]}_{folder[name]}_{subset}_{@version}<_{output}><.{@frame}><_{udim}>.{ext}'}, 'hero': {'file': '{project[code]}_{folder[name]}_{task[name]}_hero<_{comment}>.{ext}', 'folder': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/{subset}/hero', 'path': '{root[work]}/{project[name]}/{hierarchy}/{folder[name]}/publish/{family}/{subset}/hero/{project[code]}_{folder[name]}_{task[name]}_hero<_{comment}>.{ext}'}, 'delivery': {}}, 'tasks': {'Generic': {'icon': 'task_alt', 'short_name': 'gener'}, 'Art': {'icon': 'palette', 'short_name': 'art'}, 'Modeling': {'icon': 'language', 'short_name': 'mdl'}, 'Texture': {'icon': 'brush', 'short_name': 'tex'}, 'Lookdev': {'icon': 'ev_shadow', 'short_name': 'look'}, 'Rigging': {'icon': 'construction', 'short_name': 'rig'}, 'Edit': {'icon': 'imagesearch_roller', 'short_name': 'edit'}, 'Layout': {'icon': 'nature_people', 'short_name': 'lay'}, 'Setdress': {'icon': 'scene', 'short_name': 'dress'}, 'Animation': {'icon': 'directions_run', 'short_name': 'anim'}, 'FX': {'icon': 'fireplace', 'short_name': 'fx'}, 'Lighting': {'icon': 'highlight', 'short_name': 'lgt'}, 'Paint': {'icon': 'video_stable', 'short_name': 'paint'}, 'Compositing': {'icon': 'layers', 'short_name': 'comp'}, 'Roto': {'icon': 'gesture', 'short_name': 'roto'}, 'Matchmove': {'icon': 'switch_video', 'short_name': 'matchmove'}}}, 'data': {'fps': 25.0, 'resolutionWidth': 1920, 'resolutionHeight': 1080, 'pixelAspect': 1.0, 'clipIn': 1, 'clipOut': 1, 'frameStart': 1001, 'frameEnd': 1001, 'handleStart': 0, 'handleEnd': 0, 'startDate': '2023-12-25T17:43:09.758555+00:00', 'endDate': '2024-03-31T17:43:09.758568+00:00', 'entityType': 'Project', 'library_project': False, 'code': 'com', 'active': True}}"
2024-01-15 14:36:12:  0: STDOUT: DEBUG:urllib3.connectionpool:http://localhost:5000 "POST /graphql HTTP/1.1" 200 1135
2024-01-15 14:36:12:  0: STDOUT: DEBUG:GlobalServerAPI:Response <RestApiResponse [200]>
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectContextEntities:Collected Asset "{'_id': 'e2ed8484b0a811eea35a0242ac120004', 'parent': 'demo_Commercial', 'type': 'asset', 'schema': 'openpype:asset-3.0', 'name': 'sh010', 'data': {'label': 'sh010', 'entityType': 'Shot', 'visualParent': 'e2eb6b86b0a811eea35a0242ac120004', 'active': True, 'thumbnail_id': '36924ec8b3b311eea39d0242ac120004', 'parents': ['shots'], 'pixelAspect': 1.0, 'resolutionWidth': 1920, 'resolutionHeight': 1080, 'frameEnd': 1001, 'handleStart': 0, 'fps': 25.0, 'handleEnd': 0, 'clipOut': 1, 'frameStart': 1001, 'endDate': '2024-03-31T17:43:09.758568+00:00', 'description': None, 'clipIn': 1, 'startDate': '2023-12-25T17:43:09.758555+00:00', 'tools_env': None, 'tasks': {'matchmove': {'type': 'Matchmove'}, 'animation': {'type': 'Animation'}, 'fx': {'type': 'FX'}, 'lighting': {'type': 'Lighting'}, 'roto': {'type': 'Compositing'}, 'compositing': {'type': 'Compositing'}}}}"
2024-01-15 14:36:12:  0: STDOUT: DEBUG:GlobalServerAPI:Executing [GET] users/me
2024-01-15 14:36:12:  0: STDOUT: DEBUG:urllib3.connectionpool:http://localhost:5000 "GET /api/users/me HTTP/1.1" 200 360
2024-01-15 14:36:12:  0: STDOUT: DEBUG:GlobalServerAPI:Response <RestApiResponse [200]>
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectCurrentUserPype:Collected user "service"
2024-01-15 14:36:12:  0: STDOUT: DEBUG:GlobalServerAPI:Executing [GET] users/me
2024-01-15 14:36:12:  0: STDOUT: DEBUG:urllib3.connectionpool:http://localhost:5000 "GET /api/users/me HTTP/1.1" 200 360
2024-01-15 14:36:12:  0: STDOUT: DEBUG:GlobalServerAPI:Response <RestApiResponse [200]>
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectAnatomyContextData:Global Anatomy Context Data collected:
2024-01-15 14:36:12:  0: STDOUT: {
2024-01-15 14:36:12:  0: STDOUT:     "studio": {
2024-01-15 14:36:12:  0: STDOUT:         "name": "",
2024-01-15 14:36:12:  0: STDOUT:         "code": ""
2024-01-15 14:36:12:  0: STDOUT:     },
2024-01-15 14:36:12:  0: STDOUT:     "user": "service",
2024-01-15 14:36:12:  0: STDOUT:     "project": {
2024-01-15 14:36:12:  0: STDOUT:         "name": "demo_Commercial",
2024-01-15 14:36:12:  0: STDOUT:         "code": "com"
2024-01-15 14:36:12:  0: STDOUT:     },
2024-01-15 14:36:12:  0: STDOUT:     "asset": "sh010",
2024-01-15 14:36:12:  0: STDOUT:     "folder": {
2024-01-15 14:36:12:  0: STDOUT:         "name": "sh010"
2024-01-15 14:36:12:  0: STDOUT:     },
2024-01-15 14:36:12:  0: STDOUT:     "hierarchy": "shots",
2024-01-15 14:36:12:  0: STDOUT:     "parent": "shots",
2024-01-15 14:36:12:  0: STDOUT:     "task": {
2024-01-15 14:36:12:  0: STDOUT:         "name": "animation",
2024-01-15 14:36:12:  0: STDOUT:         "type": "Animation",
2024-01-15 14:36:12:  0: STDOUT:         "short": "anim"
2024-01-15 14:36:12:  0: STDOUT:     },
2024-01-15 14:36:12:  0: STDOUT:     "app": "maya",
2024-01-15 14:36:12:  0: STDOUT:     "d": "15",
2024-01-15 14:36:12:  0: STDOUT:     "dd": "15",
2024-01-15 14:36:12:  0: STDOUT:     "ddd": "Mon",
2024-01-15 14:36:12:  0: STDOUT:     "dddd": "Monday",
2024-01-15 14:36:12:  0: STDOUT:     "m": "1",
2024-01-15 14:36:12:  0: STDOUT:     "mm": "01",
2024-01-15 14:36:12:  0: STDOUT:     "mmm": "Jan",
2024-01-15 14:36:12:  0: STDOUT:     "mmmm": "January",
2024-01-15 14:36:12:  0: STDOUT:     "yy": "24",
2024-01-15 14:36:12:  0: STDOUT:     "yyyy": "2024",
2024-01-15 14:36:12:  0: STDOUT:     "H": "14",
2024-01-15 14:36:12:  0: STDOUT:     "HH": "14",
2024-01-15 14:36:12:  0: STDOUT:     "h": "2",
2024-01-15 14:36:12:  0: STDOUT:     "hh": "02",
2024-01-15 14:36:12:  0: STDOUT:     "ht": "PM",
2024-01-15 14:36:12:  0: STDOUT:     "M": "36",
2024-01-15 14:36:12:  0: STDOUT:     "MM": "36",
2024-01-15 14:36:12:  0: STDOUT:     "S": "11",
2024-01-15 14:36:12:  0: STDOUT:     "SS": "11",
2024-01-15 14:36:12:  0: STDOUT:     "username": "service"
2024-01-15 14:36:12:  0: STDOUT: }
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectContextLabel:Context label is changed to "Maya - <shell>"
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectAnatomyInstanceData:Collecting anatomy data for all instances.
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectAnatomyInstanceData:Querying asset documents for instances.
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectAnatomyInstanceData:All instances already had right asset document.
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectAnatomyInstanceData:Querying latest versions for instances.
2024-01-15 14:36:12:  0: STDOUT: DEBUG:urllib3.connectionpool:http://localhost:5000 "POST /graphql HTTP/1.1" 200 439
2024-01-15 14:36:12:  0: STDOUT: DEBUG:GlobalServerAPI:Response <RestApiResponse [200]>
2024-01-15 14:36:12:  0: STDOUT: DEBUG:urllib3.connectionpool:http://localhost:5000 "POST /graphql HTTP/1.1" 200 222
2024-01-15 14:36:12:  0: STDOUT: DEBUG:GlobalServerAPI:Response <RestApiResponse [200]>
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectAnatomyInstanceData:Storing anatomy data to instance data.
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectAnatomyInstanceData:Anatomy data for instance renderAnimationMain_beauty: {
2024-01-15 14:36:12:  0: STDOUT:     "studio": {
2024-01-15 14:36:12:  0: STDOUT:         "name": "",
2024-01-15 14:36:12:  0: STDOUT:         "code": ""
2024-01-15 14:36:12:  0: STDOUT:     },
2024-01-15 14:36:12:  0: STDOUT:     "user": "service",
2024-01-15 14:36:12:  0: STDOUT:     "project": {
2024-01-15 14:36:12:  0: STDOUT:         "name": "demo_Commercial",
2024-01-15 14:36:12:  0: STDOUT:         "code": "com"
2024-01-15 14:36:12:  0: STDOUT:     },
2024-01-15 14:36:12:  0: STDOUT:     "asset": "sh010",
2024-01-15 14:36:12:  0: STDOUT:     "folder": {
2024-01-15 14:36:12:  0: STDOUT:         "name": "sh010"
2024-01-15 14:36:12:  0: STDOUT:     },
2024-01-15 14:36:12:  0: STDOUT:     "hierarchy": "shots",
2024-01-15 14:36:12:  0: STDOUT:     "parent": "shots",
2024-01-15 14:36:12:  0: STDOUT:     "task": {
2024-01-15 14:36:12:  0: STDOUT:         "name": "animation",
2024-01-15 14:36:12:  0: STDOUT:         "type": "Animation",
2024-01-15 14:36:12:  0: STDOUT:         "short": "anim"
2024-01-15 14:36:12:  0: STDOUT:     },
2024-01-15 14:36:12:  0: STDOUT:     "app": "maya",
2024-01-15 14:36:12:  0: STDOUT:     "d": "15",
2024-01-15 14:36:12:  0: STDOUT:     "dd": "15",
2024-01-15 14:36:12:  0: STDOUT:     "ddd": "Mon",
2024-01-15 14:36:12:  0: STDOUT:     "dddd": "Monday",
2024-01-15 14:36:12:  0: STDOUT:     "m": "1",
2024-01-15 14:36:12:  0: STDOUT:     "mm": "01",
2024-01-15 14:36:12:  0: STDOUT:     "mmm": "Jan",
2024-01-15 14:36:12:  0: STDOUT:     "mmmm": "January",
2024-01-15 14:36:12:  0: STDOUT:     "yy": "24",
2024-01-15 14:36:12:  0: STDOUT:     "yyyy": "2024",
2024-01-15 14:36:12:  0: STDOUT:     "H": "14",
2024-01-15 14:36:12:  0: STDOUT:     "HH": "14",
2024-01-15 14:36:12:  0: STDOUT:     "h": "2",
2024-01-15 14:36:12:  0: STDOUT:     "hh": "02",
2024-01-15 14:36:12:  0: STDOUT:     "ht": "PM",
2024-01-15 14:36:12:  0: STDOUT:     "M": "36",
2024-01-15 14:36:12:  0: STDOUT:     "MM": "36",
2024-01-15 14:36:12:  0: STDOUT:     "S": "11",
2024-01-15 14:36:12:  0: STDOUT:     "SS": "11",
2024-01-15 14:36:12:  0: STDOUT:     "username": "service",
2024-01-15 14:36:12:  0: STDOUT:     "family": "render",
2024-01-15 14:36:12:  0: STDOUT:     "subset": "renderAnimationMain_beauty",
2024-01-15 14:36:12:  0: STDOUT:     "version": 4,
2024-01-15 14:36:12:  0: STDOUT:     "resolution_width": 1920,
2024-01-15 14:36:12:  0: STDOUT:     "resolution_height": 1080,
2024-01-15 14:36:12:  0: STDOUT:     "pixel_aspect": 1.0,
2024-01-15 14:36:12:  0: STDOUT:     "fps": 25.0
2024-01-15 14:36:12:  0: STDOUT: }
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectAnatomyInstanceData:Anatomy Data collection finished.
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectResourcesPath:publishDir: "P:\PROJECTS\demo_Commercial\shots\sh010\publish\render\renderAnimationMain_beauty\v004"
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectResourcesPath:resourcesDir: "P:\PROJECTS\demo_Commercial\shots\sh010\publish\render\renderAnimationMain_beauty\v004\resources"
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectSourceForSource:Parsing paths for {root[work]}/demo_Commercial/shots/sh010/work/animation/com_sh010_animation_v008.ma
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectCustomStagingDir:Not adding custom staging dir for instance with 'render'
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectSlackFamilies:Looking for matching profile for: families: "render" | tasks: "animation" | task_types: "Animation" | hosts: "maya" | subsets: "renderAnimationMain_beauty"
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectSlackFamilies:Profile selected: {'families': [], 'hosts': [], 'task_types': [], 'review_upload_limit': 50.0, 'channel_messages': [], 'tasks': [], 'subsets': []}
2024-01-15 14:36:12:  0: STDOUT: INFO:pyblish.CollectSlackFamilies:Found profile: {'families': [], 'hosts': [], 'task_types': [], 'review_upload_limit': 50.0, 'channel_messages': [], 'tasks': [], 'subsets': []}
2024-01-15 14:36:12:  0: STDOUT: DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 127.0.0.1:8082
2024-01-15 14:36:12:  0: STDOUT: DEBUG:urllib3.connectionpool:http://127.0.0.1:8082 "GET /api/jobs?JobID=65a5427f2e5e77f3b44c5515 HTTP/1.1" 200 3982
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ValidatePublishDir:Looking for matching profile for: hosts: "maya" | families: "render" | task_names: "animation" | task_types: "Animation"
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ValidatePublishDir:"maya" not found in "hosts": ['standalonepublisher']
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ValidatePublishDir:"render" not found in "families": ['staticMesh', 'skeletalMesh']
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ValidatePublishDir:"maya" not found in "hosts": ['traypublisher']
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ValidatePublishDir:"maya" not found in "hosts": ['max']
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ValidatePublishDir:Profile selected: {'hosts': [], 'task_types': [], 'task_names': [], 'template_name': 'publish_render', 'families': ['review', 'render', 'prerender']}
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.CollectComment:Instance renderAnimationMain_beauty does not have set comment
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.StopTimer:Publish is not affecting running timers.
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ExtractHierarchyToAYON:Skipping ExtractHierarchyToAYON
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ExtractThumbnailFromSource:Thumbnail source not filled. Skipping.
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ExtractThumbnailFromSource:Processing instance with subset name renderAnimationMain_beauty
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ExtractThumbnailFromSource:Thumbnail source not filled. Skipping.
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ExtractScanlineExr:Processing representation exr
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ExtractScanlineExr: - missing toScanline tag
2024-01-15 14:36:12:  0: STDOUT: DEBUG:GlobalServerAPI:Executing [GET] addons/ayon_third_party/1.0.0/settings?variant=dev
2024-01-15 14:36:12:  0: STDOUT: DEBUG:urllib3.connectionpool:http://localhost:5000 "GET /api/addons/ayon_third_party/1.0.0/settings?variant=dev HTTP/1.1" 200 309
2024-01-15 14:36:12:  0: STDOUT: DEBUG:GlobalServerAPI:Response <RestApiResponse [200]>
2024-01-15 14:36:12:  0: STDOUT: DEBUG:GlobalServerAPI:Executing [GET] addons/ayon_third_party/1.0.0/files_info
2024-01-15 14:36:12:  0: STDOUT: DEBUG:urllib3.connectionpool:http://localhost:5000 "GET /api/addons/ayon_third_party/1.0.0/files_info HTTP/1.1" 200 925
2024-01-15 14:36:12:  0: STDOUT: DEBUG:GlobalServerAPI:Response <RestApiResponse [200]>
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ExtractOIIOTranscode:Looking for matching profile for: hosts: "maya" | families: "render" | task_names: "animation" | task_types: "Animation" | subsets: "renderAnimationMain_beauty"
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ExtractOIIOTranscode:Profile selected: {'product_types': [], 'hosts': ['maya'], 'task_types': [], 'task_names': [], 'product_names': [], 'delete_original': True, 'outputs': {'review': {'extension': 'png', 'transcoding_type': 'colorspace', 'colorspace': 'ACES - ACEScg', 'display': '', 'view': '', 'oiiotool_args': {'additional_command_args': []}, 'tags': [], 'custom_tags': []}}}
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ExtractOIIOTranscode:repre (1): `exr`
2024-01-15 14:36:12:  0: STDOUT: DEBUG:pyblish.ExtractOIIOTranscode:Reading P:/PROJECTS/demo_Commercial/shots/sh010/work/animation/renders/maya/com_sh010_workfileAnimation_v010/Main\Main.1001.exr
2024-01-15 14:36:12:  0: STDOUT: <ImageSpec version="25">
2024-01-15 14:36:12:  0: STDOUT: <x>0</x>
2024-01-15 14:36:12:  0: STDOUT: <y>0</y>
2024-01-15 14:36:12:  0: STDOUT: <z>0</z>
2024-01-15 14:36:12:  0: STDOUT: <width>1920</width>
2024-01-15 14:36:12:  0: STDOUT: <height>1080</height>
2024-01-15 14:36:12:  0: STDOUT: <depth>1</depth>
2024-01-15 14:36:12:  0: STDOUT: <full_x>0</full_x>
2024-01-15 14:36:12:  0: STDOUT: <full_y>0</full_y>
2024-01-15 14:36:12:  0: STDOUT: <full_z>0</full_z>
2024-01-15 14:36:12:  0: STDOUT: <full_width>1920</full_width>
2024-01-15 14:36:12:  0: STDOUT: <full_height>1080</full_height>
2024-01-15 14:36:12:  0: STDOUT: <full_depth>1</full_depth>
2024-01-15 14:36:12:  0: STDOUT: <tile_width>64</tile_width>
2024-01-15 14:36:12:  0: STDOUT: <tile_height>64</tile_height>
2024-01-15 14:36:12:  0: STDOUT: <tile_depth>1</tile_depth>
2024-01-15 14:36:12:  0: STDOUT: <format>float</format>
2024-01-15 14:36:12:  0: STDOUT: <nchannels>4</nchannels>
2024-01-15 14:36:12:  0: STDOUT: <channelnames>
2024-01-15 14:36:12:  0: STDOUT: <channelname>R</channelname>
2024-01-15 14:36:12:  0: STDOUT: <channelname>G</channelname>
2024-01-15 14:36:12:  0: STDOUT: <channelname>B</channelname>
2024-01-15 14:36:12:  0: STDOUT: <channelname>A</channelname>
2024-01-15 14:36:12:  0: STDOUT: </channelnames>
2024-01-15 14:36:12:  0: STDOUT: <alpha_channel>3</alpha_channel>
2024-01-15 14:36:12:  0: STDOUT: <z_channel>-1</z_channel>
2024-01-15 14:36:12:  0: STDOUT: <deep>0</deep>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="oiio:ColorSpace" type="string">Linear</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="compression" type="string">zip</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="CameraFilmApertureHorizontal" type="float">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="CameraFilmApertureVertical" type="float">0.5625</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="CameraFocalLength" type="float">0.972222</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="CameraFov" type="float">54.4322</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/AA_samples" type="int">3</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/AA_seed" type="int">1809394299</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/aovs/RGBA/filter" type="string">gaussian_filter</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/aovs/RGBA/filter_width" type="float">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/aovs/RGBA/lpe" type="int">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/aovs/RGBA/lpe_expression" type="string">C.*</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/aovs/RGBA/source" type="string">RGBA</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/auto_transparency_depth" type="int">10</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/bounds_max_x" type="float">1919</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/bounds_max_y" type="float">1079</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/bounds_min_x" type="float">0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/bounds_min_y" type="float">0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/camera/far_clip" type="float">10000</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/camera/near_clip" type="float">0.1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/color_manager" type="string">color_manager_ocio</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/color_space" type="string">ACES - ACEScg</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/diffuse_depth" type="int">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/diffuse_samples" type="int">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/fps" type="float">25</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/frame" type="float">1001</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/host/hw" type="string">1 x Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz (4 cores, 8 logical) with 32551MB</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/host/name" type="string">DESKTOP-969549J</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/host/os" type="string">Windows 10 (version 10.0, build 19045)</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/license_state" type="string">invalid</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/specular_depth" type="int">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/specular_samples" type="int">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/sss_samples" type="int">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/date" type="string">Mon Jan 15 14:35:30 2024</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/geo/curve_segments" type="float">0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/geo/triangles" type="float">0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/memory/peak" type="float">1554.54</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/memory/start" type="float">948.727</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/rays/all/pixel" type="float">9.25155</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/rays/all/total" type="float">1.9184e+07</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/rays/camera/pixel" type="float">9.25155</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/rays/camera/total" type="float">1.9184e+07</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/time/render" type="float">8.97029</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/time/setup" type="float">13.2756</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/texture_max_memory_MB" type="float">4096</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/threads" type="int">8</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/total_depth" type="int">10</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/transmission_depth" type="int">8</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/transmission_samples" type="int">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/version" type="string">Arnold 7.0.0.3 [ccc33ba9] windows clang-10.0.1 oiio-2.3.2 osl-1.12.0 vdb-7.1.1 clm-2.0.0.235 rlm-14.1.3 optix-6.6.0 2022/02/16 18:36:18</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/volume_depth" type="int">0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/volume_samples" type="int">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="DateTime" type="string">2024:01:15 14:35:30</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="PixelAspectRatio" type="float">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="screenWindowCenter" type="float2">0, 0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="screenWindowWidth" type="float">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="worldToCamera" type="matrix">1, 0, 0, 0, 0, 1, 0, 0, 0, 0, -1, 0, 0, 0, 0, 1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="worldToScreen" type="matrix">1.94444, 0, 0, 0, 0, 3.45679, 0, 0, 0, 0, -1.00001, -1, 0, 0, -0.100001, 0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="oiio:subimages" type="int">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: </ImageSpec>
2024-01-15 14:36:12:  0: STDOUT: Reading P:/PROJECTS/demo_Commercial/shots/sh010/work/animation/renders/maya/com_sh010_workfileAnimation_v010/Main\Main.1002.exr
2024-01-15 14:36:12:  0: STDOUT: <ImageSpec version="25">
2024-01-15 14:36:12:  0: STDOUT: <x>0</x>
2024-01-15 14:36:12:  0: STDOUT: <y>0</y>
2024-01-15 14:36:12:  0: STDOUT: <z>0</z>
2024-01-15 14:36:12:  0: STDOUT: <width>1920</width>
2024-01-15 14:36:12:  0: STDOUT: <height>1080</height>
2024-01-15 14:36:12:  0: STDOUT: <depth>1</depth>
2024-01-15 14:36:12:  0: STDOUT: <full_x>0</full_x>
2024-01-15 14:36:12:  0: STDOUT: <full_y>0</full_y>
2024-01-15 14:36:12:  0: STDOUT: <full_z>0</full_z>
2024-01-15 14:36:12:  0: STDOUT: <full_width>1920</full_width>
2024-01-15 14:36:12:  0: STDOUT: <full_height>1080</full_height>
2024-01-15 14:36:12:  0: STDOUT: <full_depth>1</full_depth>
2024-01-15 14:36:12:  0: STDOUT: <tile_width>64</tile_width>
2024-01-15 14:36:12:  0: STDOUT: <tile_height>64</tile_height>
2024-01-15 14:36:12:  0: STDOUT: <tile_depth>1</tile_depth>
2024-01-15 14:36:12:  0: STDOUT: <format>float</format>
2024-01-15 14:36:12:  0: STDOUT: <nchannels>4</nchannels>
2024-01-15 14:36:12:  0: STDOUT: <channelnames>
2024-01-15 14:36:12:  0: STDOUT: <channelname>R</channelname>
2024-01-15 14:36:12:  0: STDOUT: <channelname>G</channelname>
2024-01-15 14:36:12:  0: STDOUT: <channelname>B</channelname>
2024-01-15 14:36:12:  0: STDOUT: <channelname>A</channelname>
2024-01-15 14:36:12:  0: STDOUT: </channelnames>
2024-01-15 14:36:12:  0: STDOUT: <alpha_channel>3</alpha_channel>
2024-01-15 14:36:12:  0: STDOUT: <z_channel>-1</z_channel>
2024-01-15 14:36:12:  0: STDOUT: <deep>0</deep>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="oiio:ColorSpace" type="string">Linear</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="compression" type="string">zip</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="CameraFilmApertureHorizontal" type="float">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="CameraFilmApertureVertical" type="float">0.5625</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="CameraFocalLength" type="float">0.972222</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="CameraFov" type="float">54.4322</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/AA_samples" type="int">3</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/AA_seed" type="int">1621992705</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/aovs/RGBA/filter" type="string">gaussian_filter</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/aovs/RGBA/filter_width" type="float">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/aovs/RGBA/lpe" type="int">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/aovs/RGBA/lpe_expression" type="string">C.*</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/aovs/RGBA/source" type="string">RGBA</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/auto_transparency_depth" type="int">10</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/bounds_max_x" type="float">1919</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/bounds_max_y" type="float">1079</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/bounds_min_x" type="float">0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/bounds_min_y" type="float">0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/camera/far_clip" type="float">10000</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/camera/near_clip" type="float">0.1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/color_manager" type="string">color_manager_ocio</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/color_space" type="string">ACES - ACEScg</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/diffuse_depth" type="int">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/diffuse_samples" type="int">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/fps" type="float">25</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/frame" type="float">1002</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/host/hw" type="string">1 x Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz (4 cores, 8 logical) with 32551MB</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/host/name" type="string">DESKTOP-969549J</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/host/os" type="string">Windows 10 (version 10.0, build 19045)</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/license_state" type="string">invalid</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/specular_depth" type="int">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/specular_samples" type="int">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/sss_samples" type="int">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/date" type="string">Mon Jan 15 14:35:47 2024</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/geo/curve_segments" type="float">0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/geo/triangles" type="float">0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/memory/peak" type="float">1560.13</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/memory/start" type="float">948.727</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/rays/all/pixel" type="float">9.25166</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/rays/all/total" type="float">1.91842e+07</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/rays/camera/pixel" type="float">9.25166</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/rays/camera/total" type="float">1.91842e+07</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/time/render" type="float">8.45735</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/stats/time/setup" type="float">30.7504</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/texture_max_memory_MB" type="float">4096</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/threads" type="int">8</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/total_depth" type="int">10</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/transmission_depth" type="int">8</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/transmission_samples" type="int">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/version" type="string">Arnold 7.0.0.3 [ccc33ba9] windows clang-10.0.1 oiio-2.3.2 osl-1.12.0 vdb-7.1.1 clm-2.0.0.235 rlm-14.1.3 optix-6.6.0 2022/02/16 18:36:18</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/volume_depth" type="int">0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="arnold/volume_samples" type="int">2</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="DateTime" type="string">2024:01:15 14:35:47</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="PixelAspectRatio" type="float">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="screenWindowCenter" type="float2">0, 0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="screenWindowWidth" type="float">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="worldToCamera" type="matrix">1, 0, 0, 0, 0, 1, 0, 0, 0, 0, -1, 0, 0, 0, 0, 1</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="worldToScreen" type="matrix">1.94444, 0, 0, 0, 0, 3.45679, 0, 0, 0, 0, -1.00001, -1, 0, 0, -0.100001, 0</attrib>
2024-01-15 14:36:12:  0: STDOUT: <attrib name="oiio:subimages" type="int">1</attrib>
2024-01-15 14:36:12:  0: STDOUT: </ImageSpec>
2024-01-15 14:36:12:  0: STDOUT: ERROR:pyblish.plugin:Traceback (most recent call last):
2024-01-15 14:36:12:  0: STDOUT:   File "C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2310271602_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
2024-01-15 14:36:12:  0: STDOUT:     runner(*args)
2024-01-15 14:36:12:  0: STDOUT:   File "C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\extract_color_transcode.py", line 152, in process
2024-01-15 14:36:12:  0: STDOUT:   File "C:\Users\tokejepsen\OpenPype\openpype\lib\transcoding.py", line 1136, in convert_colorspace
2024-01-15 14:36:12:  0: STDOUT:     input_info = get_oiio_info_for_input(input_path, logger=logger)
2024-01-15 14:36:12:  0: STDOUT:   File "C:\Users\tokejepsen\OpenPype\openpype\lib\transcoding.py", line 124, in get_oiio_info_for_input
2024-01-15 14:36:12:  0: STDOUT:     output.append(parse_oiio_xml_output(xml_text, logger=logger))
2024-01-15 14:36:12:  0: STDOUT:   File "C:\Users\tokejepsen\OpenPype\openpype\lib\transcoding.py", line 276, in parse_oiio_xml_output
2024-01-15 14:36:12:  0: STDOUT:     tree = xml.etree.ElementTree.fromstring(xml_string)
2024-01-15 14:36:12:  0: STDOUT:   File "xml\etree\ElementTree.py", line 1347, in XML
2024-01-15 14:36:12:  0: STDOUT: xml.etree.ElementTree.ParseError: syntax error: line 1, column 0
2024-01-15 14:36:12:  0: STDOUT: Traceback (most recent call last):
2024-01-15 14:36:12:  0: STDOUT:   File "C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2310271602_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
2024-01-15 14:36:12:  0: STDOUT:     runner(*args)
2024-01-15 14:36:12:  0: STDOUT:   File "<string>", line 152, in process
2024-01-15 14:36:12:  0: STDOUT:   File "C:\Users\tokejepsen\OpenPype\openpype\lib\transcoding.py", line 1136, in convert_colorspace
2024-01-15 14:36:12:  0: STDOUT:     input_info = get_oiio_info_for_input(input_path, logger=logger)
2024-01-15 14:36:12:  0: STDOUT:   File "C:\Users\tokejepsen\OpenPype\openpype\lib\transcoding.py", line 124, in get_oiio_info_for_input
2024-01-15 14:36:12:  0: STDOUT:     output.append(parse_oiio_xml_output(xml_text, logger=logger))
2024-01-15 14:36:12:  0: STDOUT:   File "C:\Users\tokejepsen\OpenPype\openpype\lib\transcoding.py", line 276, in parse_oiio_xml_output
2024-01-15 14:36:12:  0: STDOUT:     tree = xml.etree.ElementTree.fromstring(xml_string)
2024-01-15 14:36:12:  0: STDOUT:   File "xml\etree\ElementTree.py", line 1347, in XML
2024-01-15 14:36:12:  0: STDOUT: xml.etree.ElementTree.ParseError: syntax error: line 1, column 0
2024-01-15 14:36:12:  0: STDOUT: !!! ERR: 2024-01-15 14:36:12,405 >>> { CLI-publish }: [ Failed ExtractOIIOTranscode: syntax error: line 1, column 0 -- ('C:\\Users\\tokejepsen\\OpenPype\\openpype\\plugins\\publish\\extract_color_transcode.py', 1347, 'XML', '') ] 
2024-01-15 14:36:12:  0: STDOUT: >>> loading environments ...
2024-01-15 14:36:12:  0: STDOUT:   - global AYON ...
2024-01-15 14:36:12:  0: STDOUT:   - for addons ...
2024-01-15 14:36:12:  0: STDOUT: *** AYON [1.0.0-rc.2] 
2024-01-15 14:36:12:  0: STDOUT: >>> Using AYON from [ C:\Users\tokejepsen\AppData\Local\ynput\ayon\app\AYON 1.0.0-rc.2 ]
2024-01-15 14:36:12:  0: STDOUT: ... AYON variant: [ dev (dev) ]
2024-01-15 14:36:12:  0: STDOUT: ... AYON bundle:  [ dev ]
2024-01-15 14:36:12:  0: STDOUT: setting target: deadline
2024-01-15 14:36:12:  0: STDOUT: setting target: farm
2024-01-15 14:36:12:  0: STDOUT: Using plugins:
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_cleanup_keys.py.CollectCleanupKeys'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_current_context.py.CollectCurrentContext'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_current_shell_file.py.CollectCurrentShellFile'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_datetime_data.py.CollectDateTimeData'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_from_create_context.py.CollectFromCreateContext'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_host_name.py.CollectHostName'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_machine_name.py.CollectMachineName'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_modules.py.CollectModules'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_shell_workspace.py.CollectShellWorkspace'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_time.py.CollectTime'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_settings.py.CollectSettings'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_anatomy_object.py.CollectAnatomyObject'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_rendered_files.py.CollectRenderedFiles'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_context_entities.py.CollectContextEntities'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2310271602_windows.zip\dependencies\pyblish\plugins\collect_current_date.py.CollectCurrentDate'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2310271602_windows.zip\dependencies\pyblish\plugins\collect_current_user.py.CollectCurrentUser'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2310271602_windows.zip\dependencies\pyblish\plugins\collect_current_working_directory.py.CollectCurrentWorkingDirectory'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\repair_unicode_strings.py.RepairUnicodeStrings'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_current_pype_user.py.CollectCurrentUserPype'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_anatomy_context_data.py.CollectAnatomyContextData'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\modules\deadline\plugins\publish\collect_default_deadline_server.py.CollectDefaultDeadlineServer'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_context_label.py.CollectContextLabel'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_anatomy_instance_data.py.CollectAnatomyInstanceData'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_resources_path.py.CollectResourcesPath'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_source_for_source.py.CollectSourceForSource'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_custom_staging_dir.py.CollectCustomStagingDir'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_farm_target.py.CollectFarmTarget'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_input_representations_to_versions.py.CollectInputRepresentationsToVersions'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\modules\deadline\plugins\publish\collect_publishable_instances.py.CollectDeadlinePublishableInstances'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\AppData\Local\Ynput\AYON\addons\slack_1.0.1\ayon_slack\plugins\publish\collect_slack_family.py.CollectSlackFamilies'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\validate_asset_docs.py.ValidateAssetDocs'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\validate_filesequences.py.ValidateFileSequences'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\validate_unique_subsets.py.ValidateSubsetUniqueness'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\modules\deadline\plugins\publish\validate_deadline_pools.py.ValidateDeadlinePools'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\modules\deadline\plugins\publish\validate_expected_and_rendered_files.py.ValidateExpectedFiles'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\validate_publish_dir.py.ValidatePublishDir'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\validate_resources.py.ValidateResources'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\collect_comment.py.CollectComment'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\modules\timers_manager\plugins\publish\stop_timer.py.StopTimer'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\extract_hierarchy_avalon.py.ExtractHierarchyToAvalon'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\extract_hierarchy_to_ayon.py.ExtractHierarchyToAYON'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\extract_thumbnail_from_source.py.ExtractThumbnailFromSource'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\extract_scanline_exr.py.ExtractScanlineExr'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\extract_color_transcode.py.ExtractOIIOTranscode'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\extract_review.py.ExtractReview'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\extract_burnin.py.ExtractBurnin'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\extract_review_slate.py.ExtractReviewSlate'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\extract_colorspace_data.py.ExtractColorspaceData'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\extract_thumbnail.py.ExtractThumbnail'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\integrate_subset_group.py.IntegrateSubsetGroup'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\preintegrate_thumbnail_representation.py.PreIntegrateThumbnails'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\integrate_resources_path.py.IntegrateResourcesPath'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\integrate.py.IntegrateAsset'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\integrate_thumbnail.py.IntegrateThumbnails'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\integrate_thumbnail_ayon.py.IntegrateThumbnailsAYON'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\integrate_hero_version.py.IntegrateHeroVersion'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\integrate_inputlinks_ayon.py.IntegrateInputLinksAYON'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\AppData\Local\Ynput\AYON\addons\slack_1.0.1\ayon_slack\plugins\publish\integrate_slack_api.py.IntegrateSlackAPI'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\integrate_version_attrs.py.IntegrateVersionAttributes'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\modules\timers_manager\plugins\publish\start_timer.py.StartTimer'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\cleanup.py.CleanUp'>
2024-01-15 14:36:12:  0: STDOUT: <class 'C:\Users\tokejepsen\OpenPype\openpype\plugins\publish\cleanup_explicit.py.ExplicitCleanUp'>
2024-01-15 14:36:12:  0: STDOUT: syntax error: line 1, column 0
2024-01-15 14:36:13:  0: INFO: Process exit code: 1
2024-01-15 14:36:13:  0: Done executing plugin command of type 'Render Task'

=======================================================
Details
=======================================================
Date: 01/15/2024 14:36:16
Frames: 0
Elapsed Time: 00:00:00:15
Job Submit Date: 01/15/2024 14:34:39
Job User: tokejepsen
Average RAM Usage: 17713348608 (52%)
Peak RAM Usage: 17713348608 (52%)
Average CPU Usage: 19%
Peak CPU Usage: 26%
Used CPU Clocks (x10^6 cycles): 26359
Total CPU Clocks (x10^6 cycles): 138731

=======================================================
Worker Information
=======================================================
Worker Name: DESKTOP-969549J
Version: v10.3.0.10 Release (ac90e7082)
Operating System: Windows 10 Pro
Running As Service: No
Machine User: tokejepsen
IP Address: 192.168.196.119
MAC Address: 02:50:77:17:59:A5
CPU Architecture: x64
CPUs: 8
CPU Usage: 26%
Memory Usage: 17.7 GB / 31.8 GB (55%)
Free Disk Space: 148.998 GB (51.369 GB on C:\, 48.801 GB on G:\, 48.801 GB on H:\, 6.180 MB on K:\, 5.180 MB on L:\, 6.180 MB on P:\, 6.180 MB on T:\, 5.176 MB on Y:\)
Video Card: Intel(R) UHD Graphics
```

Reply:
Tried the code changes and it errors;

```
2024-01-15 14:39:27:  0: STDOUT: Traceback (most recent call last):
2024-01-15 14:39:27:  0: STDOUT:   File "C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2310271602_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
2024-01-15 14:39:27:  0: STDOUT:     runner(*args)
2024-01-15 14:39:27:  0: STDOUT:   File "<string>", line 152, in process
2024-01-15 14:39:27:  0: STDOUT:   File "C:\Users\tokejepsen\OpenPype\openpype\lib\transcoding.py", line 1137, in convert_colorspace
2024-01-15 14:39:27:  0: STDOUT:     input_info = get_oiio_info_for_input(input_path, logger=logger)
2024-01-15 14:39:27:  0: STDOUT:   File "C:\Users\tokejepsen\OpenPype\openpype\lib\transcoding.py", line 116, in get_oiio_info_for_input
2024-01-15 14:39:27:  0: STDOUT:     raise ValueError(
2024-01-15 14:39:27:  0: STDOUT: ValueError: Failed to read input file "P:/PROJECTS/demo_Commercial/shots/sh010/work/animation/renders/maya/com_sh010_workfileAnimation_v010/Main\Main.1001-1002#.exr".
2024-01-15 14:39:27:  0: STDOUT: Output:
2024-01-15 14:39:27:  0: STDOUT: Reading P:/PROJECTS/demo_Commercial/shots/sh010/work/animation/renders/maya/com_sh010_workfileAnimation_v010/Main\Main.1001.exr
2024-01-15 14:39:27:  0: STDOUT: <ImageSpec version="25">
2024-01-15 14:39:27:  0: STDOUT: <x>0</x>
2024-01-15 14:39:27:  0: STDOUT: <y>0</y>
2024-01-15 14:39:27:  0: STDOUT: <z>0</z>
2024-01-15 14:39:27:  0: STDOUT: <width>1920</width>
2024-01-15 14:39:27:  0: STDOUT: <height>1080</height>
2024-01-15 14:39:27:  0: STDOUT: <depth>1</depth>
2024-01-15 14:39:27:  0: STDOUT: <full_x>0</full_x>
2024-01-15 14:39:27:  0: STDOUT: <full_y>0</full_y>
2024-01-15 14:39:27:  0: STDOUT: <full_z>0</full_z>
2024-01-15 14:39:27:  0: STDOUT: <full_width>1920</full_width>
2024-01-15 14:39:27:  0: STDOUT: <full_height>1080</full_height>
2024-01-15 14:39:27:  0: STDOUT: <full_depth>1</full_depth>
2024-01-15 14:39:27:  0: STDOUT: <tile_width>64</tile_width>
2024-01-15 14:39:27:  0: STDOUT: <tile_height>64</tile_height>
2024-01-15 14:39:27:  0: STDOUT: <tile_depth>1</tile_depth>
2024-01-15 14:39:27:  0: STDOUT: <format>float</format>
2024-01-15 14:39:27:  0: STDOUT: <nchannels>4</nchannels>
2024-01-15 14:39:27:  0: STDOUT: <channelnames>
2024-01-15 14:39:27:  0: STDOUT: <channelname>R</channelname>
2024-01-15 14:39:27:  0: STDOUT: <channelname>G</channelname>
2024-01-15 14:39:27:  0: STDOUT: <channelname>B</channelname>
2024-01-15 14:39:27:  0: STDOUT: <channelname>A</channelname>
2024-01-15 14:39:27:  0: STDOUT: </channelnames>
2024-01-15 14:39:27:  0: STDOUT: <alpha_channel>3</alpha_channel>
2024-01-15 14:39:27:  0: STDOUT: <z_channel>-1</z_channel>
2024-01-15 14:39:27:  0: STDOUT: <deep>0</deep>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="oiio:ColorSpace" type="string">Linear</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="compression" type="string">zip</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="CameraFilmApertureHorizontal" type="float">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="CameraFilmApertureVertical" type="float">0.5625</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="CameraFocalLength" type="float">0.972222</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="CameraFov" type="float">54.4322</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/AA_samples" type="int">3</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/AA_seed" type="int">1809394299</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/aovs/RGBA/filter" type="string">gaussian_filter</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/aovs/RGBA/filter_width" type="float">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/aovs/RGBA/lpe" type="int">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/aovs/RGBA/lpe_expression" type="string">C.*</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/aovs/RGBA/source" type="string">RGBA</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/auto_transparency_depth" type="int">10</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/bounds_max_x" type="float">1919</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/bounds_max_y" type="float">1079</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/bounds_min_x" type="float">0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/bounds_min_y" type="float">0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/camera/far_clip" type="float">10000</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/camera/near_clip" type="float">0.1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/color_manager" type="string">color_manager_ocio</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/color_space" type="string">ACES - ACEScg</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/diffuse_depth" type="int">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/diffuse_samples" type="int">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/fps" type="float">25</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/frame" type="float">1001</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/host/hw" type="string">1 x Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz (4 cores, 8 logical) with 32551MB</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/host/name" type="string">DESKTOP-969549J</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/host/os" type="string">Windows 10 (version 10.0, build 19045)</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/license_state" type="string">invalid</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/specular_depth" type="int">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/specular_samples" type="int">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/sss_samples" type="int">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/date" type="string">Mon Jan 15 14:35:30 2024</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/geo/curve_segments" type="float">0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/geo/triangles" type="float">0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/memory/peak" type="float">1554.54</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/memory/start" type="float">948.727</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/rays/all/pixel" type="float">9.25155</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/rays/all/total" type="float">1.9184e+07</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/rays/camera/pixel" type="float">9.25155</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/rays/camera/total" type="float">1.9184e+07</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/time/render" type="float">8.97029</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/time/setup" type="float">13.2756</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/texture_max_memory_MB" type="float">4096</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/threads" type="int">8</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/total_depth" type="int">10</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/transmission_depth" type="int">8</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/transmission_samples" type="int">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/version" type="string">Arnold 7.0.0.3 [ccc33ba9] windows clang-10.0.1 oiio-2.3.2 osl-1.12.0 vdb-7.1.1 clm-2.0.0.235 rlm-14.1.3 optix-6.6.0 2022/02/16 18:36:18</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/volume_depth" type="int">0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/volume_samples" type="int">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="DateTime" type="string">2024:01:15 14:35:30</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="PixelAspectRatio" type="float">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="screenWindowCenter" type="float2">0, 0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="screenWindowWidth" type="float">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="worldToCamera" type="matrix">1, 0, 0, 0, 0, 1, 0, 0, 0, 0, -1, 0, 0, 0, 0, 1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="worldToScreen" type="matrix">1.94444, 0, 0, 0, 0, 3.45679, 0, 0, 0, 0, -1.00001, -1, 0, 0, -0.100001, 0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="oiio:subimages" type="int">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: </ImageSpec>
2024-01-15 14:39:27:  0: STDOUT: Reading P:/PROJECTS/demo_Commercial/shots/sh010/work/animation/renders/maya/com_sh010_workfileAnimation_v010/Main\Main.1002.exr
2024-01-15 14:39:27:  0: STDOUT: <ImageSpec version="25">
2024-01-15 14:39:27:  0: STDOUT: <x>0</x>
2024-01-15 14:39:27:  0: STDOUT: <y>0</y>
2024-01-15 14:39:27:  0: STDOUT: <z>0</z>
2024-01-15 14:39:27:  0: STDOUT: <width>1920</width>
2024-01-15 14:39:27:  0: STDOUT: <height>1080</height>
2024-01-15 14:39:27:  0: STDOUT: <depth>1</depth>
2024-01-15 14:39:27:  0: STDOUT: <full_x>0</full_x>
2024-01-15 14:39:27:  0: STDOUT: <full_y>0</full_y>
2024-01-15 14:39:27:  0: STDOUT: <full_z>0</full_z>
2024-01-15 14:39:27:  0: STDOUT: <full_width>1920</full_width>
2024-01-15 14:39:27:  0: STDOUT: <full_height>1080</full_height>
2024-01-15 14:39:27:  0: STDOUT: <full_depth>1</full_depth>
2024-01-15 14:39:27:  0: STDOUT: <tile_width>64</tile_width>
2024-01-15 14:39:27:  0: STDOUT: <tile_height>64</tile_height>
2024-01-15 14:39:27:  0: STDOUT: <tile_depth>1</tile_depth>
2024-01-15 14:39:27:  0: STDOUT: <format>float</format>
2024-01-15 14:39:27:  0: STDOUT: <nchannels>4</nchannels>
2024-01-15 14:39:27:  0: STDOUT: <channelnames>
2024-01-15 14:39:27:  0: STDOUT: <channelname>R</channelname>
2024-01-15 14:39:27:  0: STDOUT: <channelname>G</channelname>
2024-01-15 14:39:27:  0: STDOUT: <channelname>B</channelname>
2024-01-15 14:39:27:  0: STDOUT: <channelname>A</channelname>
2024-01-15 14:39:27:  0: STDOUT: </channelnames>
2024-01-15 14:39:27:  0: STDOUT: <alpha_channel>3</alpha_channel>
2024-01-15 14:39:27:  0: STDOUT: <z_channel>-1</z_channel>
2024-01-15 14:39:27:  0: STDOUT: <deep>0</deep>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="oiio:ColorSpace" type="string">Linear</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="compression" type="string">zip</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="CameraFilmApertureHorizontal" type="float">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="CameraFilmApertureVertical" type="float">0.5625</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="CameraFocalLength" type="float">0.972222</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="CameraFov" type="float">54.4322</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/AA_samples" type="int">3</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/AA_seed" type="int">1621992705</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/aovs/RGBA/filter" type="string">gaussian_filter</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/aovs/RGBA/filter_width" type="float">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/aovs/RGBA/lpe" type="int">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/aovs/RGBA/lpe_expression" type="string">C.*</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/aovs/RGBA/source" type="string">RGBA</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/auto_transparency_depth" type="int">10</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/bounds_max_x" type="float">1919</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/bounds_max_y" type="float">1079</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/bounds_min_x" type="float">0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/bounds_min_y" type="float">0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/camera/far_clip" type="float">10000</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/camera/near_clip" type="float">0.1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/color_manager" type="string">color_manager_ocio</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/color_space" type="string">ACES - ACEScg</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/diffuse_depth" type="int">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/diffuse_samples" type="int">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/fps" type="float">25</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/frame" type="float">1002</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/host/hw" type="string">1 x Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz (4 cores, 8 logical) with 32551MB</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/host/name" type="string">DESKTOP-969549J</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/host/os" type="string">Windows 10 (version 10.0, build 19045)</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/license_state" type="string">invalid</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/specular_depth" type="int">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/specular_samples" type="int">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/sss_samples" type="int">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/date" type="string">Mon Jan 15 14:35:47 2024</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/geo/curve_segments" type="float">0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/geo/triangles" type="float">0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/memory/peak" type="float">1560.13</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/memory/start" type="float">948.727</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/rays/all/pixel" type="float">9.25166</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/rays/all/total" type="float">1.91842e+07</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/rays/camera/pixel" type="float">9.25166</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/rays/camera/total" type="float">1.91842e+07</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/time/render" type="float">8.45735</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/stats/time/setup" type="float">30.7504</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/texture_max_memory_MB" type="float">4096</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/threads" type="int">8</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/total_depth" type="int">10</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/transmission_depth" type="int">8</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/transmission_samples" type="int">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/version" type="string">Arnold 7.0.0.3 [ccc33ba9] windows clang-10.0.1 oiio-2.3.2 osl-1.12.0 vdb-7.1.1 clm-2.0.0.235 rlm-14.1.3 optix-6.6.0 2022/02/16 18:36:18</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/volume_depth" type="int">0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="arnold/volume_samples" type="int">2</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="DateTime" type="string">2024:01:15 14:35:47</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="PixelAspectRatio" type="float">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="screenWindowCenter" type="float2">0, 0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="screenWindowWidth" type="float">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="worldToCamera" type="matrix">1, 0, 0, 0, 0, 1, 0, 0, 0, 0, -1, 0, 0, 0, 0, 1</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="worldToScreen" type="matrix">1.94444, 0, 0, 0, 0, 3.45679, 0, 0, 0, 0, -1.00001, -1, 0, 0, -0.100001, 0</attrib>
2024-01-15 14:39:27:  0: STDOUT: <attrib name="oiio:subimages" type="int">1</attrib>
2024-01-15 14:39:27:  0: STDOUT: </ImageSpec>
```

Reply:
The fact that it goes one by one through files is due to to how it's parsing the files:
```
Failed to read input file "P:/PROJECTS/demo_Commercial/shots/sh010/work/animation/renders/maya/com_sh010_workfileAnimation_v010/Main\Main.1001-1002#.exr"
```

> Do you have example output? Because this would mean that "Reading path/to/file.ext" is there after each ImageSpec, which is weird.

Which I suppose for this makes sense that it does so for each file being opened?

Reply:
> Tried the code changes and it errors;

You mean my suggested changes?

I tried this and it works as expected:
```python
output = """Reading P:/PROJECT...
...
</ImageSpec>
"""

xml_started = False
subimages_lines = []
lines = []
for line in output.split("\n"):
    if not xml_started:
        if not line.startswith("<"):
            continue
        xml_started = True

    if xml_started:
        lines.append(line)
        if line == "</ImageSpec>":
            subimages_lines.append(lines)
            lines = []
            xml_started = False

for lines in subimages_lines:
    print(lines[0])
```

Reply:
I think the output in the snippet shown above is:
```
output = """
Reading /path/to/file.1001.exr
<ImageSpec>
<width>1920</width>
<height>1080</height>
</ImageSpec>
Reading /path/to/file.1002.exr
<ImageSpec>
<width>1920</width>
<height>1080</height>
</ImageSpec>
Reading /path/to/file.1003.exr
<ImageSpec>
<width>1920</width>
<height>1080</height>
</ImageSpec>
"""
etc.